minitest-reporters-json_reporter.gemspec

```diff
@@ -19,7 +19,7 @@ See: https://atom.io. Originally written to interface with the Viper Audible edi
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '~> 2.0'
+  spec.required_ruby_version = '>= 2.0', '<= 3.0'
```

Comment:
This makes it work with 3.0, but not 3.1 or newer.

To include all of 2.x and 3.x we'd need: `'>= 2.0', '< 4.0'`